app/views/hub/state_file/efile_submissions/index.html.erb

```diff
@@ -73,8 +79,8 @@
           <td class="index-table__cell">
             <%= submission.irs_submission_id.present? ? link_to(submission.irs_submission_id, hub_state_file_efile_submission_path(id: submission.id)) : "" %>
           </td>
-          <td class="index-table__cell"><%= submission.data_source.state_name %></td>
-          <td class="index-table__cell"><%= submission.data_source.email_address %></td>
+          <td class="index-table__cell"><%= StateFileBaseIntake::STATE_CODE_AND_NAMES[submission.data_source_state_code] %></td>
+          <td class="index-table__cell"><%= submission.email_address %></td>
```

Comment:
Not using `submission.data_source.email_address` here means that we don't load the intake individually for each submission - n+1 queries becomes a single query.